app/src/main/java/com/ggetters/app/ui/central/views/StatisticsFragment.kt

```
@@ -0,0 +1,151 @@
+                }
+            }
+        }
+    }
+
+    private val viewModel: StatisticsViewModel by viewModels()
+
+    private var playerId: String? = null
+    private var currentPlayer: User? = null
+
+    // Header
+    private lateinit var closeButton: ImageButton
```

detekt (1.23.8)

[warning] Usages of lateinit should be avoided.

(detekt.potential-bugs.LateinitUsage)